### DISCUSSION TITLE
waline一直在加载，评论出不来，可能是什么问题

### DISCUSSION THREAD

**Discussion by shixiaocaia** [walinejs/waline#1123] [Q&A] [answered]
![M S FHB{((22 2ESZQQ14DE](https://user-images.githubusercontent.com/68102662/173836079-699e5abe-477a-4f24-938d-bbbd616be2f0.png)

是单纯网络卡顿吗，大部分时间很难加载出来
![{`Q5MP}MR(SYU(9 Z~RX8RF](https://user-images.githubusercontent.com/68102662/173837133-15e319c4-3ad4-4f9a-a81c-d18bd3dc243a.png)

**Comment by shixiaocaia** [comment]
![@2`P`$QMI%R}_K 4M%RQ@19](https://user-images.githubusercontent.com/68102662/173838271-a8462cd5-76f8-4f27-9c37-8b529a2e6cb0.png)
补充控制台看到的错误

**Comment by lizheming** [comment]
提供一下网站

**Reply by shixiaocaia** [reply]
https://blog-shixiaocaia.vercel.app/

**Reply by shixiaocaia** [reply]
https://blog-shixiaocaia.vercel.app/posts/04-hugo%E6%90%AD%E5%BB%BA/ 在这篇下面我发了4个评论，基本上没有成功加载过

**Comment by lizheming** [comment, marked as answer]
你的主题有做全站数学公式的渲染，这个导致 Waline 找到不到元素了。`renderMathInElement` 那块把 `document.body` 缩小到你的文章区元素就可以了。这个之前有类似的反馈。

**Reply by shixiaocaia** [reply]
已解决，与数学公式渲染确实有关
